src/main.rs

```diff
@@ -17,7 +18,8 @@ struct Args {
     #[clap(value_hint = ValueHint::FilePath, value_name = "PATH or BYTECODE")]
     pub path_or_bytecode: String,
 
-    /// Where to save the .dot string used to visualize the analyzed cfg, default is stdout
+    /// What format and location to store the analyzed cfg, default is stdout, supports all
+    /// standard graphviz formats
```

Comment:
```suggestion
    /// Filename and format for storing the analyzed cfg. Supports all standard
    /// graphviz formats (*.dot, *.png, *.jpg, ...). Default is stdout in dot format
```

adding a few common example formats for the cli hint. i wouldnt know a standard graphviz format

Reply:
this may not pass rustfmt. just did this in github webui

Reply:
lfg